.github/workflows/weekly-data-and-external-tool-updater.yml

```diff
@@ -0,0 +1,96 @@
+name: Weekly Data and External Tool Updater
+
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  checks: none
+  contents: write
```

Comment:
I think this can maybe be `read` ? 👀 

```
Work with the contents of the repository. For example, contents: read permits an action to list the commits, and contents: write allows the action to create a release
```

Let me know if that's an incorrect statement though 😄

Reply:
I know if it's `none` it results in a 503 when pushing the branch. I assumed pushing a branch was writing content